testsuite/E70.py

```diff
@@ -12,6 +12,8 @@
 del a[:]; a.append(42);
 #: E704:1:1
 def f(x): return 2
+#: E704:1:1
+async def f(x): return 2
```

Comment:
In my opinion 2 spaces is not something that should be caught. That should trigger a different warning, probably along the lines of the warnings that are triggered when there are too many spaces around a `=`.